client/jetpack-onboarding/steps/contact-form.jsx

```diff
@@ -22,7 +21,7 @@ class JetpackOnboardingContactFormStep extends React.PureComponent {
 		const subHeaderText = translate( 'Would you like to get started with a Contact Us page?' );
 
 		return (
-			<Main>
+			<div>
```

Comment:
Looks like a use case for [React 16.2's `<>`](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html), no?

Reply:
Good idea, addressed in https://github.com/Automattic/wp-calypso/pull/20596/commits/4713f064ed9fc4cdf269bca91d031ec29067e632.